aikau/src/main/resources/alfresco/services/SiteService.js

```diff
@@ -111,6 +142,20 @@ define(["dojo/_base/declare",
                { label: "create-site.dialog.type.collaboration", value: "site-dashboard" }
             ];
          }
+
+         if (this.additionalSitePresets && this.additionalSitePresets.length)
```

Comment:
it was the crude part that I didn't like; I tend to think either it's worth validating the input, or it's not, and if it's worth doing, then it's worth doing properly...